kinto/__main__.py

```diff
@@ -93,6 +99,4 @@ def main(args=None):
             pserve_argv.append('--reload')
         pserve.main(pserve_argv)
 
-
-if __name__ == "__main__":
-    main()
```

Comment:
Yes, it is hard to test and because we are using entry_points I guess we don't need it anymore.